### PR TITLE
adding game logic

### PR DESCRIPTION
**Pull request by GabrielGladykowski**
I added a game logic:
- start of the game
- inserting a circle and a cross
- blocking of busy elements
- winning
- pause after winning
- refreshment

### CODE REVIEW

**Review by MarekGluch — APPROVED** [review]

File: src/script.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      } else {
+        currentPlayer = [playerOne, playerTwo].find(
+          (player) => player.type !== currentPlayer.type
+        );
+      }
+    }
+  };
+  
+  const isWin = (boardArray, currentType) => {
+    console.log(boardArray[0]);
```
Comment (TymekGluch) [inline src/script.js:73]: unessery console.log

File: src/script.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  const round = (event) => {
+    const { target } = event;
+    if (
+      target.classList.contains(CLASSES.FIELD) &&
+      !target.classList.contains(`${CLASSES.FIELD}${TYPES.CIRCLE}`) &&
+      !target.classList.contains(`${CLASSES.FIELD}${TYPES.CROSS}`)
+    ) {
+      const [row, cell] = target.dataset.id.split('.');
+      currentBoard[row][cell] = currentPlayer.type;
+      console.log(currentBoard, isWin(currentBoard, currentPlayer.type));
```
Comment (TymekGluch) [inline src/script.js:57]: unnessery console.log

**Review by TymekGluch — CHANGES_REQUESTED** [review]

File: src/script.js (hunk 2 above)
Comment (GabrielGladykowski) [reply src/script.js:57]: I have already removed

File: src/script.js (hunk 1 above)
Comment (GabrielGladykowski) [reply src/script.js:73]: I have already removed

**Review by TymekGluch — APPROVED** [review]

**Pull request merged**